packages/video_player/video_player_android/pigeons/messages.dart

```diff
@@ -2,44 +2,51 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart = 2.9
-
-import 'package:pigeon/pigeon_lib.dart';
+import 'package:pigeon/pigeon.dart';
 
+@ConfigurePigeon(PigeonOptions(
+  dartOut: 'lib/src/messages.g.dart',
```

Comment:
I started moving to that because it makes review a bit easier if you can see that it's generated in the path name.  It's also easier when developing in an IDE to quickly distinguish between "messages.dart" and "messages.g.dart".

Reply:
That seems pretty subtle. How about `.generated.dart`? Or maybe `.pigeon.dart` as a parallel to `.mocks.dart` for the Mockito-generated code.

Reply:
".g.dart" is what is used for the JSON generator.

Reply:
Ah, okay. WFM then